### ISSUE TITLE
[Bug]: Weird file permissions of uploaded files

### ISSUE THREAD

**Issue by lkiesow** [advplyr/audiobookshelf#1426]
### Describe the issue

Uploading an audio file via the user interface will cause this file to be put on the file system marked as an executable:

```
# ls -lh --color /data/audiobooks/Lars\ Kiesow/sine/
-rwxr--r-- 1 audiobookshelf audiobookshelf 1.9M Jan 22 12:09 sine.m4a
```

Audiobookshelf should not set the executable flag for files it creates.

### Audiobookshelf version

v2.2.12

### How are you running audiobookshelf?

Debian/PPA

**Comment by advplyr** [comment]
Agree. That is happening here https://github.com/advplyr/audiobookshelf/blob/master/server/utils/filePerms.js#L90

When the UID/GID is phased out then this won't happen anymore.

**Comment by advplyr** [comment]
Fixed in [v2.2.13](https://github.com/advplyr/audiobookshelf/releases/tag/v2.2.13) with removal of UID/GID env vars
